backend/src/main/java/gov/cdc/usds/simplereport/config/FeatureFlagsConfig.java

```diff
@@ -16,28 +17,27 @@
 @EnableScheduling
 @Setter
 @Getter
+@RequiredArgsConstructor
+@Slf4j
 public class FeatureFlagsConfig {
-  @Autowired
   @Getter(AccessLevel.NONE)
   @Setter(AccessLevel.NONE)
-  private FeatureFlagRepository _repo;
+  private final FeatureFlagRepository _repo;
 
   private boolean multiplexEnabled;
+  private boolean hivEnabled;
 
   @Scheduled(fixedRateString = "60000") // 1 min
   private void loadFeatureFlagsFromDB() {
     Iterable<FeatureFlag> flags = _repo.findAll();
-    flags.forEach(
-        flag -> {
-          flagMapping(flag.getName(), flag.getValue());
-        });
+    flags.forEach(flag -> flagMapping(flag.getName(), flag.getValue()));
   }
 
   private void flagMapping(String flagName, Boolean flagValue) {
     switch (flagName) {
-      case "multiplexEnabled":
-        setMultiplexEnabled(flagValue);
-        break;
+      case "multiplexEnabled" -> setMultiplexEnabled(flagValue);
+      case "hivEnabled" -> setHivEnabled(flagValue);
```

Comment:
I see the flag is being set to be updated from the DB but I don't see any liquidbase change sets. Will this come later or will you just add the entry bastioning in to the DB?

Reply:
I think the defaults in the application yaml files should be sufficient for now, and if we need to change those values we can always bastion in.